input/fsh/extensions/ExtensionJurisdictionalFacilityIdentifier.fsh

```diff
@@ -0,0 +1,9 @@
+Extension: JurisdictionalFacilityIdentifier
+Id: Extension-jurisdictional-facility-identifier
+Title: "Jurisdictional Facility Identifier"
+Description: "An identifier associated with a birth or fetal death facility location.  Four character string per NCHS edit specifications."
+* insert ExtensionContextResource(Meta)
```

Comment:
Why does it need Meta context?

Reply:
It does not, just an oversight from copying over